src/shared/components/MSKPageDecorator/MSKPageDecorator.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+export default function (component: typeof React.Component): any {
```

Comment:
It doesn't matter much. The current decorator type restrictions are a little inconvenient. It wants the return type to be exactly the same as the parameter type.